chroma_core/lib/job.py

```diff
@@ -179,12 +186,15 @@ def invoke_agent(self, host, command, args={}):
         """
         Wrapper around AgentRpc.call which provides logging
         """
+
+        fqdn = host if is_string(host) else host.fqdn
```

Comment:
Might be easier to do:
```
fqdn = get_fqdn(host)
```

I see
```
fqdn = host if is_string(host) else host.fqdn
```
twice.

Reply:
Honestly, this should always just be a string. This is really just for compat at this point.